### PR TITLE
DB audit logging web page adjustment, synchronous modification operat…

### PR DESCRIPTION
**Pull request by ljun0712**
…ion guidance
### What is changed, added or deleted? (Required)

DB audit logging web page adjustment, synchronous modification operation guidance

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.5 (TiDB 8.5 versions)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-auditing.md [hunk 1]
```diff
@@ -24,9 +24,19 @@ The audit logging feature is disabled by default. To audit a cluster, you need t
...
+
+1. In the TiDB Cloud console, navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project.
+
+    > **Tip:**
+    >
+    > If you have multiple projects, you can click <MDSvgIcon name="icon-left-projects" /> in the lower-left corner and switch to another project.
+
+2. Click the name of your target cluster to go to its overview page, and then click **DB audit logging** in the left navigation pane.
+
+3. Database audit logging is related to cloud service providers. Therefore, the operation guide for enabling the audit logging is different for clusters deployed on different cloud service providers.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-auditing.md:39]:
```suggestion
```
Move these steps to "Step 2. Configure Amazon S3 access" and "Step 2. Configure GCS access" sections to consolidate TiDB Cloud console operations and streamline Account ID retrieval process.

File: tidb-cloud/tidb-cloud-auditing.md [hunk 2]
```diff
@@ -42,10 +52,8 @@ For more information, see [Creating a bucket](https://docs.aws.amazon.com/Amazon
 
 1. Get the TiDB Cloud account ID and the External ID of the TiDB cluster that you want to enable audit logging.
 
-    1. In the TiDB Cloud console, choose a project and a cluster deployed on AWS.
-    2. Select **Settings** > **Audit Settings**. The **Audit Logging** dialog is displayed.
-    3. In the **Audit Logging** dialog, click **Show AWS IAM policy settings**. The corresponding TiDB Cloud Account ID and TiDB Cloud External ID of the TiDB cluster are displayed.
-    4. Record the TiDB Cloud Account ID and the External ID for later use.
+    1. On the DB Audit Logging page, click the **Enable** button to enter the audit configuration page.
+    2. On the right side of the configuration page, you can get the **AWS IAM Popicy Settings**. Please record the TiDB Cloud Account ID and the External ID for later use.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-auditing.md:56]:
```suggestion

    1. In the TiDB Cloud console, navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project.

        > **Tip:**
        >
        > If you have multiple projects, you can click <MDSvgIcon name="icon-left-projects" /> in the lower-left corner and switch to another project.

    2. Click the name of your target cluster to go to its overview page, and then click **DB Audit Logging** in the left navigation pane.
    3. On the **DB Audit Logging** page, click **Enable** in the upper-right corner.
    4. In the **Enable Database Audit Logging** dialog, locate the **AWS IAM Policy Settings** section, and record **TiDB Cloud Account ID** and **TiDB Cloud External ID** for later use.
```

File: tidb-cloud/tidb-cloud-auditing.md [hunk 3]
```diff
@@ -42,10 +52,8 @@ For more information, see [Creating a bucket](https://docs.aws.amazon.com/Amazon
 
 1. Get the TiDB Cloud account ID and the External ID of the TiDB cluster that you want to enable audit logging.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-auditing.md:53]:
```suggestion
1. Get the Account ID and the External ID of the TiDB cluster that you want to enable audit logging.
```

File: tidb-cloud/tidb-cloud-auditing.md [hunk 4]
```diff
@@ -79,23 +87,23 @@ For more information, see [Creating a bucket](https://docs.aws.amazon.com/Amazon
 
 #### Step 3. Enable audit logging
 
-In the TiDB Cloud console, go back to the **Audit Logging** dialog box where you got the TiDB Cloud account ID and the External ID values, and then take the following steps:
+In the TiDB Cloud console, go back to the **Enable Database Audit Logging** dialog box where you got the TiDB Cloud account ID and the External ID values, and then take the following steps:
 
 1. In the **Bucket URI** field, enter the URI of your S3 bucket where the audit log files are to be written.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-auditing.md:92]:
```suggestion
1. In the **Bucket URL** field, enter the URI of your S3 bucket where the audit log files are to be written.
```

image

File: tidb-cloud/tidb-cloud-auditing.md [hunk 5]
```diff
@@ -111,9 +119,8 @@ For more information, see [Creating storage buckets](https://cloud.google.com/st
 
 1. Get the Google Cloud Service Account ID of the TiDB cluster that you want to enable audit logging.
 
-    1. In the TiDB Cloud console, choose a project and a cluster deployed on Google Cloud Platform.
-    2. Select **Settings** > **Audit Settings**. The **Audit Logging** dialog box is displayed.
-    3. Click **Show Google Cloud Server Account ID**, and then copy the Service Account ID for later use.
+    1. On the DB Audit Logging page, click the **Enable** button to enter the audit configuration page.
+    2. On the right side of the configuration page, you can get the **Google Cloud Server Account ID**. Please record the TiDB Cloud Account ID for later use.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-auditing.md:123]:
```suggestion
    1. In the TiDB Cloud console, navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project.

        > **Tip:**
        >
        > If you have multiple projects, you can click <MDSvgIcon name="icon-left-projects" /> in the lower-left corner and switch to another project.

    2. Click the name of your target cluster to go to its overview page, and then click **DB Audit Logging** in the left navigation pane.
    3. On the **DB Audit Logging** page, click **Enable** in the upper-right corner.
    4. In the **Enable Database Audit Logging** dialog, locate the **Google Cloud Server Account ID** section, and record **Service Account ID** for later use.
```

File: tidb-cloud/tidb-cloud-auditing.md [hunk 6]
```diff
@@ -138,22 +145,22 @@ For more information, see [Creating storage buckets](https://cloud.google.com/st
 
 #### Step 3. Enable audit logging
 
-In the TiDB Cloud console, go back to the **Audit Logging** dialog box where you got the TiDB Cloud account ID, and then take the following steps:
+In the TiDB Cloud console, go back to the **Enable Database Audit Logging** dialog box where you got the TiDB Cloud account ID, and then take the following steps:
 
 1. In the **Bucket URI** field, enter your full GCS bucket name.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-auditing.md:150]:
```suggestion
1. In the **Bucket URL** field, enter your full GCS bucket name.
```

File: tidb-cloud/tidb-cloud-auditing.md [hunk 7]
```diff
@@ -138,22 +145,22 @@ For more information, see [Creating storage buckets](https://cloud.google.com/st
...
 
-    TiDB Cloud is ready to write audit logs for the specified cluster to your Amazon S3 bucket.
+    TiDB Cloud is ready to write audit logs for the specified cluster to your GCS bucket.
 
 > **Note:**
 >
-> - After enabling audit logging, if you make any new changes to bucket URI or location, you must click **Restart** to load the changes and rerun the **Test Connectivity** check to make the changes effective.
-> - To remove GCS access from TiDB Cloud, simply delete the principal that you added.
+> - After enabling audit logging, if you make any new changes to bucket URI or location, you must click **Test Connection** to re-verify TiDB Cloud's access configuration to the bucket, and then click **Enable** for the changes to take effect.
+> - To remove TiDB Cloud's access to your GCS bucket, simply delete the trust policy granted to this cluster in your GCS bucket.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-auditing.md:163]:
```suggestion
> - After enabling audit logging, if you make any new changes to the bucket URL or location, you must click **Test Connection** again to verify that TiDB Cloud can connect to the bucket. Then, click **Enable** to apply the changes.
> - To remove TiDB Cloud's access to your GCS bucket, delete the trust policy granted to this cluster in the Google Cloud console.
```

File: tidb-cloud/tidb-cloud-auditing.md (hunk 4 above)
Comment (ljun0712) [reply tidb-cloud/tidb-cloud-auditing.md:90]: This is a wrong modification of the Staging environment. The original name "URI" is retained in the release. So keep "URI" in the document.

File: tidb-cloud/tidb-cloud-auditing.md (hunk 6 above)
Comment (ljun0712) [reply tidb-cloud/tidb-cloud-auditing.md:155]: This is a wrong modification of the Staging environment. The original name "URI" is retained in the release. So keep "URI" in the document.

File: tidb-cloud/tidb-cloud-auditing.md [hunk 8]
```diff
@@ -79,23 +85,23 @@ For more information, see [Creating a bucket](https://docs.aws.amazon.com/Amazon
...
-5. In the upper-right corner, toggle the audit setting to **On**.
+5. Click **Enable** to enable audit logging for the cluster.
 
     TiDB Cloud is ready to write audit logs for the specified cluster to your Amazon S3 bucket.
 
 > **Note:**
 >
-> - After enabling audit logging, if you make any new changes to the bucket URI, location, or ARN, you must click **Restart** to load the changes and rerun the **Test Connectivity** check to make the changes effective.
-> - To remove Amazon S3 access from TiDB Cloud, simply delete the trust policy that you added.
+> - After enabling audit logging, if you make any new changes to the bucket URL, location, or ARN, you must click **Test Connection** again to verify that TiDB Cloud can connect to the bucket. Then, click **Enable** to apply the changes.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-auditing.md:103]:
```suggestion
> - After enabling audit logging, if you make any new changes to the bucket URI, location, or ARN, you must click **Test Connection** again to verify that TiDB Cloud can connect to the bucket. Then, click **Enable** to apply the changes.
```

File: tidb-cloud/tidb-cloud-auditing.md [hunk 9]
```diff
@@ -138,22 +150,22 @@ For more information, see [Creating storage buckets](https://cloud.google.com/st
...
+4. Click **Enable** to enable audit logging for the cluster.
 
-    TiDB Cloud is ready to write audit logs for the specified cluster to your Amazon S3 bucket.
+    TiDB Cloud is ready to write audit logs for the specified cluster to your GCS bucket.
 
 > **Note:**
 >
-> - After enabling audit logging, if you make any new changes to bucket URI or location, you must click **Restart** to load the changes and rerun the **Test Connectivity** check to make the changes effective.
-> - To remove GCS access from TiDB Cloud, simply delete the principal that you added.
+> - After enabling audit logging, if you make any new changes to the bucket URL or location, you must click **Test Connection** again to verify that TiDB Cloud can connect to the bucket. Then, click **Enable** to apply the changes.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-auditing.md:167]:
```suggestion
> - After enabling audit logging, if you make any new changes to the bucket URI or location, you must click **Test Connection** again to verify that TiDB Cloud can connect to the bucket. Then, click **Enable** to apply the changes.
```

**Review by Oreoxmt — APPROVED** [review]

File: tidb-cloud/tidb-cloud-auditing.md [hunk 10]
```diff
@@ -40,12 +40,18 @@ For more information, see [Creating a bucket](https://docs.aws.amazon.com/Amazon
...
 #### Step 2. Configure Amazon S3 access
 
-1. Get the TiDB Cloud account ID and the External ID of the TiDB cluster that you want to enable audit logging.
+1. Get the Account ID and the External ID of the TiDB cluster that you want to enable audit logging.
 
-    1. In the TiDB Cloud console, choose a project and a cluster deployed on AWS.
-    2. Select **Settings** > **Audit Settings**. The **Audit Logging** dialog is displayed.
-    3. In the **Audit Logging** dialog, click **Show AWS IAM policy settings**. The corresponding TiDB Cloud Account ID and TiDB Cloud External ID of the TiDB cluster are displayed.
-    4. Record the TiDB Cloud Account ID and the External ID for later use.
+
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-auditing.md:45]:
```suggestion
```

File: tidb-cloud/tidb-cloud-auditing.md [hunk 11]
```diff
@@ -40,12 +40,17 @@ For more information, see [Creating a bucket](https://docs.aws.amazon.com/Amazon
...
-    4. Record the TiDB Cloud Account ID and the External ID for later use.
+    1. In the TiDB Cloud console, navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project.
+
+        > **Tip:**
+        >
+        > If you have multiple projects, you can click <MDSvgIcon name="icon-left-projects" /> in the lower-left corner and switch to another project.
+
+    2. Click the name of your target cluster to go to its overview page, and then click **DB Audit Logging** in the left navigation pane.
+    3. On the **DB Audit Logging** page, click **Enable** in the upper-right corner.
+    4. In the **Enable Database Audit Logging** dialog, locate the **AWS IAM Policy Settings** section, and record **TiDB Cloud Account ID** and **TiDB Cloud External ID** for later use.
```
Comment (tennix) [inline tidb-cloud/tidb-cloud-auditing.md:53]: Here we use **TiDB Cloud Account ID**, but in line 43 you changed "TiDB Cloud account ID" to "Account ID". I think users will get confused by these two terms.

File: tidb-cloud/tidb-cloud-auditing.md [hunk 12]
```diff
@@ -40,12 +40,17 @@ For more information, see [Creating a bucket](https://docs.aws.amazon.com/Amazon
 
 #### Step 2. Configure Amazon S3 access
 
-1. Get the TiDB Cloud account ID and the External ID of the TiDB cluster that you want to enable audit logging.
+1. Get the Account ID and the External ID of the TiDB cluster that you want to enable audit logging.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-auditing.md:43]:
```suggestion
1. Get the TiDB Cloud Account ID and the External ID of the TiDB cluster that you want to enable audit logging.
```

File: tidb-cloud/tidb-cloud-auditing.md (hunk 11 above)
Comment (Oreoxmt) [reply tidb-cloud/tidb-cloud-auditing.md:53]: Fix in https://github.com/pingcap/docs/pull/19448/commits/3773172f7ec07ed132e639fde084c608ca44d8b4
```diff
- 1. Get the Account ID and the External ID of the TiDB cluster that you want to enable audit logging.
+ 1. Get the TiDB Cloud Account ID and the External ID of the TiDB cluster that you want to enable audit logging.
```

**Pull request merged**